README.md

```diff
@@ -25,7 +25,14 @@ Agent Learning Framework (ALF) is a reinforcement learning framework emphasizing
 
 ## Installation
 
-Python3.7 is currently supported by ALF. [Virtualenv](https://virtualenv.pypa.io/en/latest/) is recommended for the installation. After activating a virtual env, you can run the following commands to install ALF
+Python3.7 is currently supported by ALF. [Virtualenv](https://virtualenv.pypa.io/en/latest/) is recommended for the installation.
+
+Make sure python3.7-dev is installed before creating an virtual enivronment:
+```
+apt install -y python3.7-dev
```

Comment:
"sudo apt install" What's the difference between libpython3.7-dev and python3.7-dev?

Reply:
Added sudo.